primitives/io/src/lib.rs

```diff
@@ -123,7 +142,10 @@ pub trait Storage {
 	///
 	/// Returns the SCALE encoded hash.
 	fn root(&mut self) -> Vec<u8> {
-		self.storage_root()
+		let id = PROFILER.lock().create_span(module_path!(), "root");
+		let res = self.storage_root();
+		PROFILER.lock().exit_span(id);
```

Comment:
Why? These calls are already tracked automatically.

Reply:
yes, this is left over from before and doesn't have to be there, will remove